.github/workflows/build_test.yml

```diff
@@ -279,6 +279,7 @@ jobs:
         env:
           PYTHON_VERSION: ${{ matrix.python-version }}
           cloud_provider: ${{ matrix.cloud-provider }}
+          PYTEST_ADDOPTS: --color=yes
```

Comment:
What is this for?

Reply:
It allows py.test to use colored output. Previously I set this in our tox file, but that also made Jenkins try to use colored output. Jenkins cannot deal with that, so I moved the setting into GH actions. Now we can keep colored output here and not have it on Jenkins.